triton-server/src/contact-service.ts

```diff
@@ -173,7 +177,11 @@ function getCredentialMessageFor(contact: Contact) {
         <br/>
         Endpoint: <b>mcgilluniversity#genomecentre-lims</b><br/>
         Username: <b>${contact.project_id}</b><br/>
-        Password: <b>${contact.depth ?? "ERROR COULD NOT GENERATE PASSWORD"}</b><br/><br/>
+        Password: <b>${contact.depth ?? "ERROR COULD NOT FIND PASSWORD"}</b><br/>
+        <br/>
+        Using Globus: <a href="${usingGlobus}">${usingGlobus}</a><br/>
+        <b>Please unlink the identity after you're done:</b> <a href="${unlinkIdentity}">${unlinkIdentity}</a><br/>
```

Comment:
"Technical limitations prevent us from allowing more than one user to have access to the data at the same time. In order to allow other members of your team to access data, please unlink your identity from the data once you are done with the transfer. Here is a page to help with that operation : ... ."